source/jsonnet/england-wales/individual/blocks/identity-and-health/arrive_in_country.jsonnet

```diff
@@ -48,11 +48,17 @@ function(region_code, census_month_year_date) {
   routing_rules: [
     {
       goto: {
-        block: 'when-arrive-in-uk',
+        block: 'length-of-stay',
+        when: [rules.under1],
+      },
+    },
```

Comment:
Shouldn't it be more explicit?:
```
      goto: {
        block: 'length-of-stay',
        when: [
          {
            id: 'arrive-in-country-answer',
            condition: 'equals',
            value: 'not set',
          },
          rules.under1,
        ],
      },
```

Reply:
I did this because if you are under 1 then it does not make sense to go to `when-arrive-in-uk` even if they have answered the question. If you are under 1, then arrive in country will always be after census date since the difference is calculated is based on census date. Have confirmed this is the expected behaviour with Donna.

Reply:
Ok, thanks.